docs/conf.py

```diff
@@ -6,24 +6,31 @@
 # full list see the documentation:
 # http://www.sphinx-doc.org/en/master/config
 
+from __future__ import unicode_literals
+
+import os
+import sys
+import logging
```

Comment:
This can be removed.

Reply:
i need os and sys for the path setup part, i just moved them upwards to be with the others. the logging i will remove though

Reply:
Yeah, the comment was just on the `import logging` line.